Multisig/Features/Security/ProtectedKeyStore.swift

```diff
@@ -179,7 +179,7 @@ class ProtectedKeyStore: EncryptedStore {
         )
         try store.create(pubKeyItem)
 
-        if locked {
+        if locked && self.protectionClass != .data {
```

Comment:
Maybe it would be better to pass an unlock override so `ProtectedKeystore` doesn't need to behave differently by store kind?

Reply:
I think this is a case specific solution, I would go with more generic solution like passing the exception as parameter

Reply:
It is not really the exception. It is happening in 50% of the cases as long as we have two stores (data & sensitive).

Reply:
Also `protectionClass` has to be public in `ProtectedStore`, or we pass this flag from a place where we know which store type it is.

Reply:
The more I think about this, I think this decision **should** be made in the ProctectedKeyStore based on the protection class. So it is not possible to do this for the sensitive key store.